packages/node_modules/@ciscospark/internal-plugin-board/test/unit/spec/board.js

```diff
@@ -239,6 +239,25 @@ describe(`plugin-board`, () => {
     });
   });
 
+  describe(`#deletePartialContent()`, () => {
+
+    before(() => {
+      spark.request.reset();
+      return spark.internal.board.deletePartialContent(channel, [data1]);
```

Comment:
I missed this before, sorry. The function under test should generally be called in the test method, rather than before method. This test probably doesn't need a before hook.